packages/plugins/plugin-mermaid/src/extensions/mermaid.ts

```diff
@@ -155,6 +152,6 @@ const styles = EditorView.baseTheme({
   // },
   '& .cm-mermaid-error': {
     display: 'inline-block',
-    color: getToken('extend.colors.red.500'),
+    color: 'var(--dx-error)',
```

Comment:
I meant to remove/deduplicate/qualify this `error` token with the rest in `messageValence.ts` — I think the scope of this PR could, maybe should, be extended to handle that too.

Reply:
Maybe not, though, this PR seems to just address CM and that’s fine, can remove the bare `error` token in another pass that handles `messageValence`.